manipulate_pim_data/rule/general_information_on_rule_format.rst

```diff
@@ -282,7 +282,7 @@ Two parameters are required, two other are optional.
 Remove
 ______
 
-This action removes values to a multiselect, a category or a collection.
+This action removes values to a multiselect or a product category.
```

Comment:
This action removes values *from* a multiselect attribute or a product category.